### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Damjan-Bulatovic-EY**

### CODE REVIEW

**Comment by Milica-Roljic-EY** [comment]
Damjan, please make additional changes described below to improve styling and best code practices.

Component folders should be capitalized too, not only components.
Please update name to Carousel component instead of Card, to be semantically correct. 
You can create Card component as child component where you'll pass props from parent component Carousel. Card component should return jsx containing img, h3 as card title, and p to display card description (explanation).
Carousel should be centered.
Carousel cards have different image sizes, which makes carousel description text to overlap, update styling to have uniform styling.
Footer should be sticky.
Screenshot 2021-06-28 145707
Screenshot 2021-06-28 144651

UseFetch hook should be placed in separate folder and that folder could be moved to separate folder, e.g. for utils, service, helpers, or simply hooks, since it is not actually a component.
Please move key for NASA API to separate file, since it is not a good practice to have it hardcoded in the code.
Some alt attributes are added, which is great, but not in all places, fix where is missing.
Fix console warning Warning: Can't perform a React state update on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
    at Card (http://localhost:3000/static/js/main.chunk.js:688:64)
    at div
    at Home
Missing bullets (dots) - let's try to add it after updating the PR with above mentioned changes. 
The Router can go to the separate file. (optional)

**Review by Milica-Roljic-EY — CHANGES_REQUESTED** [review]
Damjan, please make changes added in comment in Conversation tab.

File: src/components/Card/Card.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  const reducedExplanation = singleData.explanation
+    .split(" ")
+    .slice(0, 10)
+    .join(" ");
+
+  return (
+    <div className="card">
+      <img src={singleData.url} className="image" alt="description" />
+      <div className="text">
+        <h3>{singleData.title}</h3>
```
Comment (mitrovic-jovana) [inline src/components/Card/Card.js:14]: This is a good place to use destructuring of singleData object

File: src/components/Carusel/Carusel.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
+import "./Carusel.scss";
+import useFetch from "../../hooks/useFetch";
+import Card from "../Card/Card";
+import leftArrow from "../../img/arrow left.svg";
+import rightArrow from "../../img/arrow right.svg";
+
+function Carusel() {
+  const key = process.env.REACT_APP_API_KEY;
```
Comment (mitrovic-jovana) [inline src/components/Carusel/Carusel.js:9]: Good job of using .env file for api keys

File: src/components/Carusel/Carusel.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from "react";
```
Comment (mitrovic-jovana) [inline src/components/Carusel/Carusel.js:1]: This component should be named **Carousel**, you have a typo

**Review by mitrovic-jovana — APPROVED** [review]
Good job Damjan, you definitely followed provided suggestions. 
I am merging this PR, however, try to spend some time fixing the design and UX a little bit. For example:
1.	Navigation arrows could be restyled, they are really small and hard to notice. Don’t put them directly next to the window edges
2.	Description in the carousel is still overflowing. Remove fixed height and add some padding
3.	You can add some spinner while you are waiting for the service response
4.	Since you don’t have pagination, one suggestion for better UX is to disable left arrow when you are at the beginning of the carousel and right arrow when you get to the end of carousel.
5.	Static pages (Terms and Conditions and Privacy Notice) could have some margin
6.	Work on adding pagination

One more thing you can improve, don’t invoke NASA’s service directly inside Carousel component. If you do that, Carousel is not customizable, and it can only work with response from that api. Invoke service in parent component and pass data to Carousel as props

**Pull request merged**